### PR TITLE
refactor: Pay the debt of some TODO comments

### PR DESCRIPTION
**Pull request by tassoevan**

### CODE REVIEW

File: apps/meteor/client/components/GenericTable/GenericTable.tsx [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
+export const GenericTable = forwardRef(function GenericTable(
+	{ fixed = true, children, ...props }: GenericTableProps,
+	ref: ForwardedRef<HTMLElement>,
+) {
 	return (
 		<Box mi='neg-x24' pi={24} flexShrink={1} flexGrow={1} ref={ref} overflow='hidden'>
 			<ScrollableContentWrapper overflowX>
-				{/* TODO: Fix fuselage */}
-				<Table fixed={fixed} sticky {...(props as any)}>
+				{' '}
```
Comment (dougfabris) [inline apps/meteor/client/components/GenericTable/GenericTable.tsx:18]: why?

File: apps/meteor/client/components/TextCopy.tsx [hunk 2]
```diff
@@ -41,7 +38,7 @@ const TextCopy = ({ text, wrapper = defaultWrapperRenderer, ...props }: TextCopy
 			{...props}
 		>
 			<Scrollable vertical>{wrapper(text)}</Scrollable>
-			<Button icon='copy' secondary square small flexShrink={0} onClick={onClick} title={t('Copy')} />
+			<Button icon='copy' secondary square small flexShrink={0} onClick={handleClick} title={t('Copy')} />
```
Comment (dougfabris) [inline apps/meteor/client/components/TextCopy.tsx:41]: why not: 
```suggestion
			<Button icon='copy' secondary square small flexShrink={0} onClick={() => copy()} title={t('Copy')} />
```

File: apps/meteor/client/components/connectionStatus/ConnectionStatusBar.tsx [hunk 3]
```diff
@@ -1,54 +1,14 @@
...
-
-	return reconnectCountdown;
-};
-
-const ConnectionStatusBar: FC = function ConnectionStatusBar() {
+const ConnectionStatusBar = function ConnectionStatusBar() {
 	const { connected, retryTime, status, reconnect } = useConnectionStatus();
 	const reconnectCountdown = useReconnectCountdown(retryTime, status);
-	const t = useTranslation();
+	const { t } = useTranslation();
```
Comment (dougfabris) [inline apps/meteor/client/components/connectionStatus/ConnectionStatusBar.tsx:11]: I think its not going to work here, in this file

**Review by dougfabris — CHANGES_REQUESTED** [review]

File: apps/meteor/client/components/connectionStatus/ConnectionStatusBar.tsx (hunk 3 above)
Comment (tassoevan) [reply apps/meteor/client/components/connectionStatus/ConnectionStatusBar.tsx:11]: Why not?

Comment (dougfabris) [reply apps/meteor/client/components/connectionStatus/ConnectionStatusBar.tsx:11]: I think there's a verification on the provider and just translations with the proper pattern works with `react-i18next`

**Review by dougfabris — APPROVED** [review]
LGTM!

**Pull request merged**